testcases/baitap2.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Library     SeleniumLibrary
+Library     OperatingSystem
+Library     Collections
+
+Suite Setup       Run Keywords      Navigate To Homepage
+Suite Teardown    Exit Selenium
+
+*** Variables ***
+${SiteURL}    http://live.techpanda.org/index.php/customer/account/login/
+${Browser}    Chrome
+${LOCATOR1}    //h2[@class='product-name']//a[normalize-space()='Sony Xperia']
+${LOCATOR2}    //p[@class='product-name']//a[normalize-space()='Sony Xperia']
+
+*** Keywords ***
+Navigate To Homepage
+    Open Browser               ${SiteURL}    ${BROWSER}
+    Maximize Browser Window
+
+Compare text
+    ${text1}=    Get Text    ${LOCATOR1}
+    ${text2}=    Get Text    ${LOCATOR2}    
+    IF    '${text1}' == '${text2}'
+            Log    The product has been added to cart
+    ELSE
+            Log    The product has not been added to cart
+    END
```

Comment:
Log không thể hiện được case pass hay fail nếu không kiểm tra file log.
Nên sử dụng Should Be Equal: https://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Should%20Be%20Equal